04.bowling/bowling.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+
+score_strings = score.split(',')
+
+scores = []
+score_strings.each do |n|
+  if n == 'X'
+    scores << 10
+    scores << 0
+  else
+    scores << n.to_i
+  end
+end
+
+frames = scores.each_slice(2).to_a
+
+frame_with_number = {}
+frames.each.with_index(1) { |fp, i| frame_with_number.store(i, fp) }
+
+total =
+  frame_with_number.each.sum do |number_of_frame, points|
```

Comment:
```suggestion
  frame_with_number.each.sum do |frame_number, points|
```

もしくは

```suggestion
  frame_with_number.each.sum do |frame_no, points|
```

とかでいいかも。

変数やメソッドの名前ってあまりofって使わないんですよね（まったくないわけではないが、あまり出てこない）。

Reply:
>変数やメソッドの名前ってあまりofって使わないんですよね（まったくないわけではないが、あまり出てこない）。

そうなのですね！
代替案までご提示いただきありがとうございます🙇‍♀️
ofを使用しない名前に修正いたしました！